frontend/src/metabase/search/components/SearchFilterModal/SearchFilterModal/SearchFilterModal.tsx

```diff
@@ -83,14 +98,14 @@ export const SearchFilterModal = ({
         </Flex>
         {availableFilters.map(key => {
           const Filter = filterMap[key];
-          return (
+          return Filter ? (
```

Comment:
I'm going to separate this logic into a different component in the next filter PR and make this logic look a little cleaner